composer.json

```diff
@@ -24,15 +24,14 @@
         "symfony/console": "^5.1.8",
         "symfony/event-dispatcher-contracts": "^2.2",
         "yiisoft/di": "@dev",
-        "yiisoft/friendly-exception": "^1.0",
```

Comment:
https://github.com/yiisoft/yii-console/blob/b828bae2cf1c5ee3a559f888ca7cd81216efb67f/src/Application.php#L13 is used.

Reply:
It wasn't removed, just comma deletion :)